samza-api/src/main/java/org/apache/samza/operators/windows/WindowKey.java

```diff
@@ -26,14 +26,19 @@
  *
  */
 public class WindowKey<K> {
-
+  /**
+   * A (key,paneId) tuple uniquely identifies an emission from a window. For instance, in case of keyed-tumbling time windows,
+   * the key is provided by the keyExtractor function, and the paneId is the start of the time window boundary. In case
+   * of session windows, the key is provided by the keyExtractor function, and the paneId is the time at which the earliest
```

Comment:
Yes that's right. I can also document it. The two are essentially equivalent in case of session windows.